content/en/blog/_posts/2023-11-02-mid-cycle-1.29.md

```diff
@@ -0,0 +1,81 @@
+
+For more information about how to enable and run external cloud controller managers, please see the Kubernetes documentation [Cloud Controller Manager Administration](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/) and [Migrate Replicated Control Plane To Use Cloud Controller Manager](https://kubernetes.io/docs/tasks/administer-cluster/controller-manager-leader-migration/).
+
+For general information about cloud controller managers, please see the Kubernetes document
+[Cloud Controller Manager](https://kubernetes.io/docs/concepts/architecture/cloud-controller/).
+
+### Removal of `v1beta2` flow control API group
+
+The flowcontrol.apiserver.k8s.io/v1beta2 API version of FlowSchema and PriorityLevelConfiguration will no longer be [served in v1.29](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-29). 
+
+To prepare for this, you can edit your existing manifests and rewrite client software to use the `flowcontrol.apiserver.k8s.io/v1beta3` API version, available since v1.26. All existing persisted objects are accessible via the new API. Notable changes in `flowcontrol.apiserver.k8s.io/v1beta3` include
+the PriorityLevelConfiguration spec.limited.assuredConcurrencyShares field is renamed to spec.limited.nominalConcurrencyShares.
+ 
+
+### Deprecation of `status.nodeInfo.kubeProxyVersion` field of v1.Node
+
+The `.status.kubeProxyVersion` field for Node objects will be [marked as deprecated](https://github.com/kubernetes/enhancements/issues/4004) in v1.29 in preparation for its removal in a future release. This field is not accurate and is set by kubelet, which does not actually know the kube-proxy version, or even if kube-proxy is running.
+
+### Deprecation of the `SkipReadOnlyValidationGCE` feature gate
+
+The `SkipReadOnlyValidationGCE` feature gate will be [deprecated](https://github.com/kubernetes/kubernetes/pull/121083). Previous versions of Kubernetes on Google Cloud required that workloads (for example Deployments, DaemonSets, etc.) which used PersistentDisk volumes were using them in read-only mode. This validation provided very little value at a relatively high host implementation cost, and will no longer be validated. If this is a problem for a specific use-case, you'll be able to the set SkipReadOnlyValidationGCE gate to false so as to re-enable the validation, and then we hope you would file a Kubernetes bug with details. 
```

Comment:
This needs tech review - it doesn't make sense to me as it stands. I think the change is to _graduate_ the feature gate.

Reply:
This doesn't make sense to include, IMO.  It's suuuuuuuch a tiny thing. It's not even that something which used to work will now fail. Things that used to fail will still fail, just but differently.

Reply:
ok, we can remove it. Thanks for the feedback @thockin!

Reply:
Thanks for the review! It was resolved in this [commit](https://github.com/kubernetes/website/pull/43439/commits/e814c6c7508cc4c2e5990c421d7b6cf252b6adb8)